src/data/battle_moves.h

```diff
@@ -2763,9 +2797,9 @@ const struct BattleMove gBattleMoves[MOVES_COUNT] =
     [MOVE_CONVERSION_2] =
     {
         #if B_UPDATED_MOVE_DATA >= GEN_5
-            .target = MOVE_TARGET_USER,
+            .target = MOVE_TARGET_SELECTED,
         #else
-            .target = MOVE_TARGET_FOES_AND_ALLY,
+            .target = MOVE_TARGET_USER,
         #endif
         .effect = EFFECT_CONVERSION_2,
         .power = 0,
```

Comment:
Conversion 2 might need some research into whether adjustments to its effect are required based on the target difference. 

Also, Accuracy should be changed to 0 (and it should be 0 for Gen 3/4 since it never misses due to self targeting; it's misleading in the move info otherwise).